satpy/dataset/metadata.py

```diff
@@ -27,33 +27,38 @@
 from satpy.writers.utils import flatten_dict
 
 
-def combine_metadata(*metadata_objects, average_times=True):
+def combine_metadata(*metadata_objects):
     """Combine the metadata of two or more Datasets.
 
     If the values corresponding to any keys are not equal or do not
     exist in all provided dictionaries then they are not included in
-    the returned dictionary.  By default any keys with the word 'time'
-    in them and consisting of datetime objects will be averaged. This
-    is to handle cases where data were observed at almost the same time
-    but not exactly.  In the interest of time, lazy arrays are compared by
-    object identity rather than by their contents.
+    the returned dictionary.
+
+    All values of the keys containing the substring 'start_time' will be set
+    to the earliest value and similarly for 'end_time' to latest time.  All
+    other keys containing the word 'time' are averaged.  Before these adjustments,
+    `None` values resulting from data that don't have times associated to them
+    are removed. These rules are applied also to values in the 'time_parameters'
+    dictionary.
+.. versionchanged:: 0.47
+   Before Satpy 0.47, all times, including `start_time` and `end_time`, were averaged.
+    In the interest of processing time, lazy arrays are compared by object
+    identity rather than by their contents.
 
```

Comment:
Looks like newlines and indentation didn't translate from the github comment. I think this is as I would expected it. Everything in the same indentation for the main docstring level then the versionchanged description is 3 spaces further indented (to align with the "v" in "versionchanged". I *think* that works. I don't remember between markdown and rst if it prefers 3 or 4 spaces for these.


```suggestion
    dictionary.
    
    .. versionchanged:: 0.47

       Before Satpy 0.47, all times, including `start_time` and `end_time`, were averaged.

    In the interest of processing time, lazy arrays are compared by object
    identity rather than by their contents.

```

Reply:
Thanks, I'll generate the docs locally tomorrow to see it looks correct.

Reply:
It seems that anything from one extra space works. I'll keep it at three for visual alignment.